pkg/git/Util.go

```diff
@@ -41,34 +43,22 @@ const (
 
 func GetLocationForMaterial(material *sql.GitMaterial) (location string, err error) {
 	//gitRegex := `/(?:git|ssh|https?|git@[-\w.]+):(\/\/)?(.*?)(\.git)(\/?|\#[-\d\w._]+?)$/`
-	gitRegex := `^https.*`
-	matched, err := regexp.MatchString(gitRegex, material.Url)
-	if matched {
-		location := strings.ReplaceAll(material.Url, "https://", "")
-		checkoutPath := path.Join(GIT_BASE_DIR, strconv.Itoa(material.Id), location)
+	httpsRegex := `^https.*`
```

Comment:
different method used in ci-runner why? better to keep it same

Reply:
also make it constant

Reply:
This method has different purpose. 
This method returns the location on which repo would be cloned. git-sensor supports multitanency for its single machine. (ci-runner runs in sandbox environment. this logic makes no sense there.)